src/components/dialogs/connectivity/connectivity-form.jsx

```diff
@@ -75,40 +75,42 @@ export const ConnectivityForm = ({
     });
 
     useEffect(() => {
-        if (watchVoltageLevelId) {
-            const voltageLevelTopologyKind = voltageLevelOptions.find(
-                (vl) => vl.id === watchVoltageLevelId
-            )?.topologyKind;
-            switch (voltageLevelTopologyKind) {
-                case 'NODE_BREAKER':
-                    fetchBusbarSectionsForVoltageLevel(studyUuid, currentNodeUuid, watchVoltageLevelId).then(
-                        (busbarSections) => {
-                            setBusOrBusbarSectionOptions(busbarSections);
-                        }
-                    );
-                    break;
+        if (!isEquipmentModification) {
```

Comment:
Since it is a pre-condition, in my opinion we can do as following
```ts
        if (isEquipmentModification) {
            return;
        } 
```